packages/config/config/devices/0x0234/zif5028_heatit_z-din_616.json

```
@@ -15,7 +16,104 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "RELAY OUTPUT. Basic Report On/Off when relay output is activated.",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set On / Off Input 1",
+			"description": "Sends Basic Set On / Off when input 1 is activated.",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Binary Switch Set On / Off Input 1",
+			"description": "Sends Binary Switch Set On / Off when input 1 is activated.",
```

```suggestion
			"label": "Input 1: Binary Switch Set",
```
etc. for the next ones too